app/cdap/components/FooterPanel/index.tsx

```diff
@@ -0,0 +1,181 @@
+  rowCount: number;
+  columnCount: number;
+}
+
+interface IFooterPanelProps {
+  recipeStepsCount: number;
+  gridMetaInfo: IGridMetaInfo;
+}
+
+const ColumnViewBox = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 8px 32px;
+  gap: 8px;
+  width: 88px;
+  height: 40px;
+  border-left: 1px solid #3994ff66;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: linear-gradient(180deg, #4681f400 0.85%, #4681f433 118.78%);
+  border-right: 1px solid #3994ff66;
+  cursor: pointer;
+`;
+
+const DirectivesBox = styled(Box)`
+  text-align: center;
+  gap: 8px;
+  width: 9.5%;
+  height: 40px;
+  background: linear-gradient(180deg, #4681f400 0.85%, #4681f433 118.78%);
+  border-left: 1px solid rgba(57, 148, 255, 0.4);
+  cursor: pointer;
+`;
+
+const Label = styled(Typography)`
+  line-height: 40px;
+`;
+
+const ReciepeStepsBox = styled(Box)`
+  text-align: center;
+  padding: 9.5px 12px;
+  gap: 8px;
+  width: 13.5%;
+  height: 40px;
+  background: linear-gradient(180deg, #4681f400 0.85%, #4681f433 118.78%);
+  border-left: 1px solid rgba(57, 148, 255, 0.4);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const TabsWrapper = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: space-between;
+  background-color: #f3f6f9;
+  height: 40px;
+  box-shadow: 0px -2px 2px #0000001a;
+  width: 100%;
+  position: absolute;
+  bottom: 54px;
+`;
+
+const TransformatedIconButton = styled(IconButton)`
+  transform: rotate(90deg);
+`;
+
+const OutlinedLabel = styled(Label)`
+  background-color: ${grey[600]};
+  line-height: 21px;
+  width: 20px;
+  color: #ffffff;
+  border-radius: 4px;
+`;
+
+const ZoomBox = styled(Box)`
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  width: 10.5%;
+  height: 40px;
+  background: linear-gradient(180deg, #4681f400 0.85%, #4681f433 118.78%);
+  border-left: 1px solid rgba(57, 148, 255, 0.4);
+  cursor: pointer;
+`;
+
+export interface ITableMetaInfoTabProps {
+  rowCount: number;
+  columnCount: number;
+}
+
+const LargeBox = styled(Box)`
+  width: 65%;
+  padding: 0px 32px;
+`;
+
+export interface IRecipeStepsTabProps {
+  recipeStepsCount: number;
+}
+
+export default function ({ recipeStepsCount, gridMetaInfo }: IFooterPanelProps) {
+
+  const { rowCount, columnCount } = gridMetaInfo;
+
+  return (
+    <TabsWrapper data-testid="footer-panel-wrapper">
+      <CustomTooltip title={`${T.translate(`${PREFIX}.columnViewPanel`)}`}>
+        <ColumnViewBox data-testid="footer-panel-column-view-panel-tab">{ColumnIcon}</ColumnViewBox>
+      </CustomTooltip>
+      <LargeBox data-testid="footer-panel-meta-info-tab">
+        <Label data-testid="footerpanel-simple-label">
+          <>
+            {`${T.translate(`${PREFIX}.currentData`)} 
+          - ${rowCount} ${T.translate(`${PREFIX}.rows`)} ${T.translate(
```

Comment:
Please do not construct I18n strings piecemeal - this should be a single I18n string of format `Current data - {{rowCount}} rows and {{columnCount}} columns`

Reply:
Updated as mentioned above